ApplicationLibCode/ProjectDataModel/RimEclipseView.cpp

```diff
@@ -456,7 +456,7 @@ void RimEclipseView::onCreateDisplayModel()
     else if ( this->cellResult()->hasStaticResult() || this->cellEdgeResult()->hasResult() ||
               this->eclipsePropertyFilterCollection()->hasActiveFilters() ||
               this->intersectionCollection()->hasAnyActiveSeparateResults() ||
-              this->surfaceInViewCollection()->hasAnyActiveSeparateResults() )
+              ( this->surfaceInViewCollection() && this->surfaceInViewCollection()->hasAnyActiveSeparateResults() ) )
```

Comment:
Should the other collections (intersection, eclipsePropertyFilter) also be checked for nullptr?

Reply:
Other collections are ok